config.js

```diff
@@ -0,0 +1,39 @@
+
+require('dotenv').config();
+// In this file you can configure migrate-mongo
+let dbName, dbUrl = '';
+if (process.env.NODE_ENV === 'test') {
+  dbName = 'cfh_test';
+  dbUrl = process.env.MONGOHQ_URL_TEST;
+} else if (process.env.NODE_ENV === 'development') {
+  dbName = 'cfh';
+  dbUrl = process.env.MONGOHQ_URL;
+} else if (process.env.NODE_ENV === 'staging') {
+  dbName = 'gondor-cfh-staging';
+  dbUrl = process.env.MONGOHQ_URL_STAGING;
+} else {
+  dbName = 'gondor-cfh';
+  dbUrl = process.env.MONGOHQ_URL_PROD;
+}
+
```

Comment:
Isn't the `dbName` also conveyed in the `dbUrl`'s value? For example, assuming the`dbUrl` is `mongodb://localhost:27017/gondor_cfh` the last part of this url has the `dbName` included. Is there any particular reason why we need to specify the `dbName`?

Reply:
Yes there's a reason. The npm module I used to facilitate migration "migrate-mongo" requires that the dbName value be supplied even if the url already contains the database name. I've tried finding a way to work around it, all to no avail.